content/en/getting_started/integrations/aws.md

```diff
@@ -105,13 +105,13 @@ Before getting started, ensure you have the following prerequisites:
 
 2. Go to the [AWS integration configuration page][8] in Datadog and click **Add AWS Account**.
 
-3. Configure the integration's settings under the **Automatically using CloudFormation** option.  
-    a. Select the AWS regions to integrate with.  
-    b. Add your Datadog [API key][9].  
-    c. Optionally, send logs and other data to Datadog with the [Datadog Forwarder Lambda][1].  
+3. Configure the integration's settings under the **Automatically using CloudFormation** option.
+    a. Select the AWS regions to integrate with.
+    b. Add your Datadog [API key][9].
+    c. Optionally, send logs and other data to Datadog with the [Datadog Forwarder Lambda][1].
```

Comment:
```suggestion
3. Configure the integration's settings under the **Automatically using CloudFormation** option.  
    a. Select the AWS regions to integrate with.  
    b. Add your Datadog [API key][9].  
    c. Optionally, send logs and other data to Datadog with the [Datadog Forwarder Lambda][1].  
```

Reply:
These spaces are necessary for Markdown formatting in this case. Screenshot of how this appears without spaces, from the [preview](https://docs-staging.datadoghq.com/gus/agent-link-update/getting_started/integrations/aws/#setup):
https://a.cl.ly/rRuzbZ4Q